server.php

```diff
@@ -150,7 +179,11 @@ function valet_default_site_path($config)
 );
 
 if (! $frontControllerPath) {
-    show_valet_404();
+    if (isset($valetConfig['directory-listing']) && $valetConfig['directory-listing'] == 'on') {
```

Comment:
I think you can skip the `else` since `show_directory_listing` exits.

Reply:
But, then the directory-listing switch would have no purpose, right?

Reply:
No, I mean do it like this:

```php
    if (isset($valetConfig['directory-listing']) && $valetConfig['directory-listing'] == 'on') {
        show_directory_listing($valetSitePath, $uri);
    }

    show_valet_404();
```

Reply:
Oh. Yes, of course. 👍